sample-apps/radiology/main.py

```diff
@@ -219,6 +219,7 @@ def main():
     conf = {
         "models": "deepedit",
         "use_pretrained_model": "false",
+        "network": "unetr",
```

Comment:
The model is huge.. making it default is not a good idea..
And these config will be still part of config/deepedit..

Refer config/segmentation on how to add config per model

Reply:
I guess this part of main function.. example to run inference.. in that case.. its fine..

Reply:
Thanks for your comments ... I'll change this bit